integration/src/test/scala/org/knora/webapi/core/LayersTest.scala

```diff
@@ -85,56 +84,16 @@ object LayersTest {
     pekko.actor.ActorSystem & AppConfigurationsTest & JwtConfig & WhichSipiService
 
   type CommonR =
-    AdminApiEndpoints &
-      AdminModule.Provided &
-      ApiRoutes &
-      ApiV2Endpoints &
-      AppRouter &
-      AssetPermissionsResponder &
-      Authenticator &
-      AuthorizationRestService &
-      CardinalityHandler &
-      ConstructResponseUtilV2 &
-      DspIngestClient &
-      GravsearchTypeInspectionRunner &
-      GroupRestService &
-      HttpServer &
-      IIIFRequestMessageHandler &
-      InferenceOptimizationService &
-      InvalidTokenCache &
-      IriConverter &
-      JwtService &
-      ListsResponder &
-      ListsResponderV2 &
-      MessageRelay &
-      OntologyCache &
-      OntologyHelpers &
-      OntologyInferencer &
-      OntologyRepo &
-      OntologyResponderV2 &
-      PermissionRestService &
-      PermissionUtilADM &
-      PermissionsResponder &
-      ProjectExportService &
-      ProjectExportStorageService &
-      ProjectImportService &
-      ProjectRestService &
-      QueryTraverser &
-      RepositoryUpdater &
-      ResourceUtilV2 &
-      ResourcesResponderV2 &
-      RestCardinalityService &
-      SearchApiRoutes &
-      SearchResponderV2 &
-      SipiService &
-      StandoffResponderV2 &
-      StandoffTagUtilV2 &
-      State &
-      StringFormatter &
-      TestClientService &
-      TriplestoreService &
-      UserRestService &
-      ValuesResponderV2
+    AdminModule.Provided & AdminApiEndpoints & ApiRoutes & ApiV2Endpoints & AppRouter & AssetPermissionsResponder &
+      Authenticator & AuthorizationRestService & CardinalityHandler & ConstructResponseUtilV2 & DspIngestClient &
+      GravsearchTypeInspectionRunner & GroupRestService & HttpServer & IIIFRequestMessageHandler &
+      InferenceOptimizationService & InvalidTokenCache & IriConverter & JwtService & ListsResponder & ListsResponderV2 &
+      MessageRelay & OntologyCache & OntologyHelpers & OntologyInferencer & OntologyRepo & OntologyResponderV2 &
+      PermissionRestService & PermissionUtilADM & PermissionsResponder & ProjectExportService &
+      ProjectExportStorageService & ProjectImportService & ProjectRestService & QueryTraverser & RepositoryUpdater &
+      ResourceUtilV2 & ResourcesResponderV2 & RestCardinalityService & SearchApiRoutes & SearchResponderV2 &
+      SipiService & StandoffResponderV2 & StandoffTagUtilV2 & State & StringFormatter & TestClientService &
+      TriplestoreService & UserRestService & ValuesResponderV2
```

Comment:
nitpick: I preferred these to be on separate lines for friendlier diffs

Reply:
I tried finding the scalafmt rule, it's a bit hard to google for, I tried a few ```binPack.parentConstructors```, but that doesn't help. It's also slow to reformat, which doesn't help.

Reply:
If scalafmt supports it I'd be happy.

Reply:
First one to find the option wins a medal (chocolate).

Reply:
I think I found something, not the neatest but might be sufficient for all these multilines - https://scalameta.org/scalafmt/docs/configuration.html#-format-off
I have the issues with building the project in this branch, could you please try it out @seakayone or I will continue to check that after lunch. Postpone the merge pelase.

Reply:
I confirm, on-off works. Could you please revert formatting in all places where classes were groups and apply above rule? Thanks!

Reply:
done

Reply:
Great, thank you!
@siers did I just win the chocolate? :D

Reply:
You do!